src/wp-includes/rest-api/class-wp-rest-server.php

```diff
@@ -1068,7 +1068,7 @@ public function dispatch( $request ) {
 	 * @return bool Whether the REST server is currently handling a request.
 	 */
 	public function is_dispatching() {
-		return empty( $this->dispatching_requests );
+		return ! empty( $this->dispatching_requests );
```

Comment:
Since the property is registered, `return (bool) $this->dispatching_requests;` would also work (and is [about 15% faster](https://3v4l.org/P4PHh)).

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/5658/commits/e09b4a0c1d05b9ecd812c2ce0f5a07a300910f5e